bundles/org.openhab.binding.plugwiseha/src/main/java/org/openhab/binding/plugwiseha/internal/handler/PlugwiseHABridgeHandler.java

```diff
@@ -175,12 +175,16 @@ private void run() {
         } catch (PlugwiseHAUnauthorizedException | PlugwiseHANotAuthorizedException e) {
             updateStatus(OFFLINE, CONFIGURATION_ERROR, STATUS_DESCRIPTION_INVALID_CREDENTIALS);
         } catch (PlugwiseHACommunicationException e) {
+            this.logger.trace("Bridge encountered an error {} {}", e.getMessage(), e.getStackTrace());
```

Comment:
To log stack trace:
`this.logger.trace("Bridge encountered an error {}", e.getMessage(), e);`
Same for others.

Reply:
Fixed